src/read.h

```diff
@@ -64,6 +71,11 @@ class Read {
 	int sample_id;
 	int id;
 	int reference_start;
+	std::string chromosome;
+	std::string read_subAlignment_id;
+	bool is_supplementary;
+	int reference_end;
+	bool is_reverse;
```

Comment:
I’m not so keen on adding all these attributes to the `Read` class. I’ll have to think about this more, but an alternative is to track this information separately in a separate data structure (and only in `haplotag.py`).